back/app/models/custom_field.rb

```diff
@@ -274,7 +274,8 @@ def printable?
     build_in_types = %w[text_multiloc html_multiloc image_files files topic_ids]
     ideation_types = ParticipationMethod::Ideation::ALLOWED_EXTRA_FIELD_TYPES
     native_survey_types = ParticipationMethod::NativeSurvey::ALLOWED_EXTRA_FIELD_TYPES
-    all_input_types = build_in_types + ideation_types + native_survey_types
+    user_field_types = %w[checkbox date] # Only when 'user_fields_in_form' is enabled
+    all_input_types = build_in_types + ideation_types + native_survey_types + user_field_types
```

Comment:
Not functional criticism, but wouldn't `built_in_types` be grammatically better?